src/Illuminate/Foundation/Console/EnvironmentEncryptCommand.php

```diff
@@ -22,7 +22,8 @@ class EnvironmentEncryptCommand extends Command
                     {--cipher= : The encryption cipher}
                     {--env= : The environment to be encrypted}
                     {--prune : Delete the original environment file}
-                    {--force : Overwrite the existing encrypted environment file}';
+                    {--force : Overwrite the existing encrypted environment file}
+                    {--interactive : Prompt the encryption key}';
```

Comment:
Should just use `$this->input->isInteractive()` instead of adding new option. https://github.com/symfony/console/blob/7.1/Input/InputInterface.php#L125

Reply:
Did not know the existence of this one. Have used that instead. However, I've then also added a `select` prompt since I do want the interactive user to be able to opt for a generated key instead of having to provide one. This will however change the current interactive command line experience. Not sure if that would be a problem.